src/test/java/build/buildfarm/cas/cfc/CASFileCacheTest.java

```diff
@@ -1108,6 +1110,8 @@ public void newInputThrowsNoSuchFileExceptionWithoutDelegate() throws Exception
             /* maxEntrySizeInBytes=*/ 1024,
             /* hexBucketLevels=*/ 1,
             storeFileDirsIndexInMemory,
+            false,
+            false,
```

Comment:
provide the /* paramName=*/ helpers here and above

Reply:
added